recipes/libtool/all/conanfile.py

```diff
@@ -57,14 +59,22 @@ def _configure_autotools(self):
         conf_args = [
             "--datarootdir={}".format(datarootdir),
             "--prefix={}".format(prefix),
-            "--disable-shared",
-            "--disable-static",
+            "--enable-shared",
+            "--enable-static",
             "--disable-ltdl-install",
         ]
         self._autotools.configure(args=conf_args, configure_dir=self._source_subfolder)
         return self._autotools
 
+    def _patch_sources(self):
+        # FIXME: tools.patch cannot patch read-only files on Windows
```

Comment:
Also, this is no longer needed, Conan 1.24 uses patch-ng 1.17.4 (https://github.com/conan-io/python-patch-ng/releases/tag/1.17.4)

Reply:
Indeed. Removed these lines.